config/compose_download_conf.sh

```diff
@@ -14,6 +14,7 @@ export CONFSTORAGE_GOGADGET=${CONFSTORAGE}gogadget/
 export CONFSTORAGE_OBJDUMP=${CONFSTORAGE}objdump/
 export CONFSTORAGE_PEID=${CONFSTORAGE}peid/
 export CONFSTORAGE_PEINFO=${CONFSTORAGE}peinfo/
+export CONFSTORAGE_SHODAN=${CONFSTORAGE}shodan/
```

Comment:
Actually thinking about this again, we should probably sort this one by alphabet, so it would be nice if you could move the passivetotal entry to the right position